docs-source/content/faq/boot-identity-not-valid.md

```diff
@@ -10,20 +10,18 @@ description: "One or more WebLogic Server instances in my domain will not start
 >
 > ***<Feb 6, 2020 12:05:35,550 AM GMT> <Critical> <Security> <BEA-090402> <Authentication denied: Boot identity not valid. The user name or password or both from the boot identity file (boot.properties) is not valid. The boot identity may have been changed since the boot identity file was created. Please edit and update the boot identity file with the proper values of username and password. The first time the updated boot identity file is used to start the server, these new values are encrypted.>***
 
-When you see these kinds of errors, it means that the user name and password provided in the `weblogicCredentialsSecret` are incorrect. Prior to operator version 2.5.0, this error could
-have also indicated that the WebLogic domain directory's security configuration files have changed in an incompatible way between when the operator scanned
+When you see these kinds of errors, it usually means that the user name and password provided in the `weblogicCredentialsSecret` are incorrect. Prior to operator version 2.5.0, this error could have also indicated that the WebLogic domain directory's security configuration files have changed in an incompatible way between when the operator scanned
 the domain directory, which occurs during the "introspection" phase, and when the server instance attempted to start. There is now a separate validation for that condition described in the [Domain secret mismatch](../domain-secret-mismatch/) FAQ entry.
 
 Check that the user name and password credentials stored in the Kubernetes Secret referenced by `weblogicCredentialsSecret` contain the expected values for an account with administrative privilege for the WebLogic domain.
```

Comment:
Kubernetes Secret referenced by `weblogicCredentialsSecret` contain -> Kubernetes Secret, referenced by `weblogicCredentialsSecret`, contain (needs commas for readability)